test/array-buffer-slow.js

```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line no-use-extend-native/no-use-extend-native
+delete ArrayBuffer.prototype.resize;
+
+// eslint-disable-next-line import/no-unassigned-import, ava/no-import-test-files, import/first
+import './array-buffer.js';
```

Comment:
We want to repeat all the tests related to `getStreamAsArrayBuffer()` but emulating an environment where `ArrayBuffer.resize()` does not exist. This is one quick way to achieve this.